samples/boards/96b_argonkey/microphone/README.rst

```diff
@@ -85,6 +85,10 @@ characters 'cooked' out.
    stty -F /dev/ttyUSB0 115200 raw
    cat /dev/ttyUSB0 > /tmp/sound.raw
 
+.. note:: The "cat /dev/ttyUSB0 > /tmp/sound.raw" command should be launched after the audio
```

Comment:
formatting looks odd and not aligned.

Reply:
@nashif 
Not sure I got your point. Can you be more specific?
The note I added is exactly the same as the other one. Why is "not aligned"?

Reply:
@nashif 
ping

Reply:
@nashif 
Can you tell me what your doubt is?
I think that this doc change is required.

Reply:
was expecting be:

```
.. note:

   <note>
```

but I did not notice the double :: after note, so it is good. Sorry

Reply:
ok, thx